Task_TheFlexGame/index.css

```diff
@@ -1 +1,85 @@
-/* Start coding below for FLEX GAME TASK*/
+.container{
+    width: 2828px;
+    height: 2562px;
+    display: flex;
+    justify-content: space-between;
+}
+.menu{
+    width: 639px;
+    height: 2562px;
+    background: #604AC6;
+    display: flex;
+    flex-direction: column;

```

Comment:
Consider adding units to the padding value for clarity (e.g., padding: 84.5px;).